modules/claims_api/app/controllers/claims_api/v2/blueprints/intent_to_file_blueprint.rb

```diff
@@ -5,10 +5,17 @@ module V2
     module Blueprints
       class IntentToFileBlueprint < Blueprinter::Base
         identifier :id
-        field :creation_date
-        field :expiration_date
-        field :status
-        field :type
+        field :type do |_itf, _options|
+          'intent_to_file'
+        end
+        field :attributes do |itf, _options|
+          {
+            creation_date: itf[:creation_date],
+            expiration_date: itf[:expiration_date],
+            type: ClaimsApi::V2::IntentToFile::ITF_TYPES_TO_BGS_TYPES.index(itf[:type]),
+            status: itf[:status].downcase
```

Comment:
we could potentially break this in to another blueprinter and create an association.

these also match v1's layout - that was my goal with adding these.

Reply:
one exception* v1 gets returned in snakecase..... the transform below returns it as camelcase. I would assume we'd want camelcase.